manifests/config.pp

```diff
@@ -24,7 +24,7 @@
 
   user { $foreman_proxy::user:
     ensure  => 'present',
-    shell   => '/bin/false',
+    shell   => $::foreman_proxy::params::shell,
```

Comment:
Since init inherits from params, the ```::params``` is not needed and would allow for easier overriding in the future.